packages/preact-iso/hydrate.d.ts

```diff
@@ -0,0 +1,3 @@
+import { VNode } from 'preact';
+
+export default function hydrate(jsx: VNode, parent?: VNode): void;
```

Comment:
Curious if this should be `ComponentChildren` instead of `VNode` similar to our types in core: https://github.com/preactjs/preact/blob/b1a42a1bc61bf1f648e19a41cdf54d235810714c/src/index.d.ts#L228-L231